.bumpversion.cfg

```diff
@@ -1,12 +1,12 @@
 [bumpversion]
-current_version = 1.1.0rc1
+current_version = 1.1.0rc2
 parse = (?P<major>\d+)
 	\.(?P<minor>\d+)
 	\.(?P<patch>\d+)
 	((?P<prekind>a|b|rc)
 	(?P<pre>\d+)  # pre-release version num
 	)?
-serialize =
+serialize = 
```

Comment:
This might fail the code quality check. Not sure why space got added

Reply:
Yep it failed, I think this is added by the GHA that bumps the version